rules-java-diva/tests/src/test/java/org/jboss/windup/rules/apps/diva/DivaTest.java

```diff
@@ -0,0 +1,93 @@
+package org.jboss.windup.rules.apps.diva;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.apache.commons.collections4.IterableUtils;
+import org.apache.commons.io.FileUtils;
+import org.apache.tinkerpop.gremlin.process.traversal.P;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.forge.arquillian.AddonDependencies;
+import org.jboss.forge.arquillian.archive.AddonArchive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.windup.exec.WindupProcessor;
+import org.jboss.windup.exec.configuration.WindupConfiguration;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.GraphContextFactory;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.rules.apps.java.config.SourceModeOption;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.locationtech.jts.util.Assert;
+
+import org.jboss.windup.rules.apps.diva.model.DivaAppModel;
+import org.jboss.windup.rules.apps.diva.model.DivaOpModel;
+import org.jboss.windup.rules.apps.diva.model.DivaTxModel;
+
+@RunWith(Arquillian.class)
+public class DivaTest {
+
+    @Deployment
+    @AddonDependencies
+    public static AddonArchive getDeployment() {
+        return ShrinkWrap.create(AddonArchive.class).addClass(DivaTest.class).addBeansXML();
+    }
+
+    @Inject
+    private WindupProcessor processor;
+
+    @Inject
+    private GraphContextFactory factory;
+
+    @Test
+    public void testSpring() throws IOException {
+        try (GraphContext context = factory.create(true)) {
+            Path inputPath = Paths.get("src/test/resources/spring/");
+            Path outputPath = Paths.get(FileUtils.getTempDirectory().toString(), "Windup")
+                    .resolve(UUID.randomUUID().toString());
+
+            WindupConfiguration windupConfiguration = new WindupConfiguration().setGraphContext(context)
+                    .setOptionValue(SourceModeOption.NAME, true).addInputPath(inputPath).setOutputDirectory(outputPath);
```

Comment:
I was expecting a new option (e.g. `enableDiva`) for enabling Diva but I can not see anything like this in tests.
Is it going to be added?

Reply:
I will follow your suggestion, e.g., option's name, default value, etc.

Reply:
During the last meeting we decided together to use the `enableTransactionsReport` CLI option for enabling the Diva transactions report generation.